controller/search.go

```diff
@@ -286,15 +286,19 @@ func (c *SearchController) Spaces(ctx *app.SpacesSearchContext) error {
 		q = "" // Allow empty query if * specified
 	}
 
+	sort, err := space.ParseSortSpaceBy(ctx.Sort)
+	if err != nil {
+		return err
+	}
+
 	var result []space.Space
 	var count int
-	var err error
 
 	offset, limit := computePagingLimits(ctx.PageOffset, ctx.PageLimit)
 
 	return application.Transactional(c.db, func(appl application.Application) error {
 		var resultCount uint64
-		result, resultCount, err = appl.Spaces().Search(ctx, &q, &offset, &limit)
+		result, resultCount, err = appl.Spaces().Search(ctx, &q, &offset, &limit, sort)
```

Comment:
Is sorting applied before the result is paginated? Just want to ensure the the entire search result is sorted and not just the paged (default 20) items returned.

Reply:
yep it is